backend/settings/settings.py

```diff
@@ -374,3 +374,18 @@
 VIDEO_METADATA_EXPIRE_SECONDS = 2 * 24 * 3600  # 2 days
 
 RECOMMENDATIONS_MIN_CONTRIBUTORS = 2
+
+# Configuration of the app `core`
+# See the documentation for the complete description.
+APP_CORE = {
+    "MGMT_DELETE_INACTIVE_USERS_PERIOD":
+        server_settings.get(
+            "CORE_DELETE_INACTIVE_USERS_PERIOD",
+            # if no configuration has been provided, align the value with the
+            # validity of the validation emails.
+            REST_REGISTRATION.get(
+                "REGISTER_VERIFICATION_PERIOD",
+                7
+            )
```

Comment:
`REST_REGISTRATION.REGISTER_VERIFICATION_PERIOD` accepts a `timedelta` value:
https://django-rest-registration.readthedocs.io/en/latest/detailed_configuration/register.html#register-verification-period

So overriding the default value in `REST_REGISTRATION` would break this configuration.

Reply:
Thanks, good catch.

Reply:
Fixed by https://github.com/tournesol-app/tournesol/pull/799/commits/c3ffa21b16e6e562503fc2f73f6886711b0c14ed.